pkg/apis/crd/v1alpha1/types.go

```diff
@@ -20,6 +20,234 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+type TraceflowPhase string
+
+const (
+	Pending   TraceflowPhase = "Pending"
```

Comment:
seems not a strong convention, there are other cases too: https://github.com/kubernetes/kubernetes/blob/72da0b1bb06607f3f3e067f1bb5ce329ec861e1b/pkg/apis/core/types.go#L504-L530. 
This patch adds "Component" because "NetworkPolicy" is the name of a struct too.